libraries/adaptive-expressions/src/builtinFunctions/startOfDay.ts

```diff
@@ -53,11 +54,9 @@ export class StartOfDay extends ExpressionEvaluator {
      */
     private static evalStartOfDay(timeStamp: string, format?: string): ValueWithError {
         let result: string;
-        const { value: parsed, error: parseError } = InternalFunctionUtils.parseTimestamp(timeStamp);
-        let error = parseError;
+        const error = InternalFunctionUtils.verifyISOTimestamp(timeStamp);
         if (!error) {
-            const startOfDay = moment(parsed).utc().hours(0).minutes(0).second(0).millisecond(0);
-            ({ value: result, error } = InternalFunctionUtils.returnFormattedTimeStampStr(startOfDay, format));
+            result = dayjs(timeStamp).utc().startOf('d').format(format);
```

Comment:
Can we use `'day'` instead of `'d'`? Just a bit more obvious at first glance.

Reply:
Sure. Change all the shorthand with the full name to make it more readable.